pages/home.md

```diff
@@ -44,10 +44,12 @@ permalink: /
           <div class="column is-offset-1 is-5">
             <h2 class="title">Your pod: All your data, under your control</h2>
             <p class="content is-large">
-            <a href="{{site.baseurl}}/about">Solid</a> lets people store their 
-            data securely in decentralized data stores 
-            called <b>Pods</b>. Pods are like secure personal web servers for data.
-            People control who and what can access data stored in their Pods.</p>
+              <a href="{{site.baseurl}}/about">Solid</a> lets you store your 
```

Comment:
Since the title is "Your pod: All your data, under your control",  updated the subject in the paragraph to "You" instead of "People".